src/List.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class List extends React.PureComponent {
+  state = {
+    todo: this.props.todo,
+    users: this.props.user,
+    checkboxStatus: false,
+  }
+
+  handlerCB = () => {
+    this.setState(({ checkboxStatus }) => ({
+      checkboxStatus: !checkboxStatus,
+    }));
+  }
+
+  render() {
+    const executor = (this.state.users
+      .filter(user => user.id === this.state.todo.userId))[0].name;
+
+    return (
+      <>
+        <div className="list__item">
+          <p className="list__item_task">{this.state.todo.title}</p>
+          <div className={this.state.checkboxStatus
+            ? 'list__item_completed'
+            : 'list__item_completed-not'}
```

Comment:
there is a super convenient classnames lib that you can use in such cases https://www.npmjs.com/package/classnames

Reply:
fixed